rfc/9/index.md

```diff
@@ -0,0 +1,342 @@
+- Include all `zarr.json` files at the beginning of the file and at the beginning of the central directory header in a breadth-first order, starting with the root-level `zarr.json` as the first entry. This enables efficient metadata processing and discovery of the hierarchy structure.
+- Include an OME-Zarr-specific archive comment in the ZIP file header, indicating compliance with these recommendations. This further facilitates efficient data/metadata access and also allows for additional (optional/recommended) single-file metadata that may be specified in future OME-Zarr versions.
+
+This RFC explicitly prohibits embedding a zipped OME-Zarr file as subhierarchy of a parent OME-Zarr hierarchy.
+In particular this prohibits "recursive zipping", the embedding of a zipped OME-Zarr file within a parent zipped OME-Zarr file.
+This restriction may be revised in the future, especially in the light of the "collections" RFC.
+
+Furthermore, this RFC prohibits splitting up the ZIP archive into multiple files ("multi-volume archives"), in favor of directory-backed OME-Zarr and Zarr's sharding codec.
+
+Finally, this RFC also defines a new file extension to be used specifically with zipped OME-Zarr files.
+This should enable file type detection (in absence of a magic number), improve user experience (e.g. by enabling file type association), avoid "accidental" in-place extraction (e.g. using on-board tooling of some operating systems) and encourage the use of OME-Zarr-specific tooling for creating zipped OME-Zarr files (to follow the recommendations listed earlier).
+
+Note that this RFC does not - semantically or otherwise - restrict the data content of OME-Zarr hierarchies to be stored in zipped OME-Zarr files.
+
+## Specification
+
+Amend the specification with the following section:
+
+### Single-file OME-Zarr
+
+This section specifies how to store an OME-Zarr hierarchy within a single file.
+
+#### Zipped OME-Zarr files
+
+An OME-Zarr hierarchy MAY be stored within a ZIP archive.
+
+For a ZIP file to be referred to as a single-file OME-Zarr, it MUST contain exactly one OME-Zarr hierarchy.
+The root of the ZIP archive MUST correspond to the root of the OME-Zarr hierarchy, and it MUST contain the the OME-Zarr's root-level `zarr.json`.
+
+When creating zipped OME-Zarr files, the following are RECOMMENDED:
+
+- The ZIP64 format extension SHOULD be used to allow for files larger than 4 GiB.
+- ZIP-level compression SHOULD be disabled in favor of Zarr-level compression codecs.
+- The sharding codec SHOULD be used to reduce the number of entries within the ZIP archive.
+- The root-level `zarr.json` file SHOULD be the first ZIP file entry and the first entry in the central directory header; other `zarr.json` files SHOULD follow immediately afterwards, in breadth-first order.
+
+These recommendations are REQUIRED for a zipped OME-Zarr file specialization from here on referred to as "recommended variant".
```

Comment:
```suggestion
These recommendations are REQUIRED for a "OME-Zarr zip file".
```

While there may be zip files containing Zarr hierarchies with OME-metadata they are not OME-Zarr zip files.